dracut/modules.d/99asahi-firmware/install-asahi-firmware.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+
+type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+info ":: Asahi: Installing vendor firmware to root filesystem..."
+if [ ! -d /sysroot/lib/firmware/vendor ]; then
+    warn ":: Asahi: Vendor firmware directory missing on the root filesystem!"
+    return 1
+fi
+mount -t tmpfs vendorfw /sysroot/lib/firmware/vendor
+cp -r /vendorfw/* /vendorfw/.vendorfw.manifest /sysroot/lib/firmware/vendor
```

Comment:
Better to keep timestamps here to avoid confusion

```suggestion
cp -pr /vendorfw/* /vendorfw/.vendorfw.manifest /sysroot/lib/firmware/vendor
```

Reply:
Fixed.